CMakeLists.txt

```
@@ -3,6 +3,11 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/MSVC.cmake
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 project(V8 LANGUAGES CXX C ASM)
 
+if(WIN32)
```

```suggestion
if(MSVC)
```
At least, I assume `ASM_MASM` shouldn't be used when targeting e.g. clang.